doc/modules/calibration.rst

```diff
@@ -56,21 +60,22 @@ by showing the number of samples in each predicted probability bin.
 
 .. currentmodule:: sklearn.linear_model
 
-:class:`LogisticRegression` returns well calibrated predictions by default as it directly
-optimizes :ref:`log_loss`. In contrast, the other methods return biased probabilities;
-with different biases per method:
+:class:`LogisticRegression` returns well calibrated predictions by default as it is
+build the canonical link function for its loss, i.e. :ref:`log_loss`. This leads to
```

Comment:
```suggestion
:class:`LogisticRegression` returns well-calibrated predictions by default as it is
built with the canonical link function for its loss, i.e. :ref:`log_loss`. This leads to
```

Reply:
I don't know if we could be succinctly didactic to give a way to find out what a canonical link function is for users that are not necessarily used to the GLMs

Reply:
Maybe a hyperlink redirecting to the GLM user guide is enough.